lib/install/bin/yarn.tt

```diff
@@ -6,5 +6,6 @@ Dir.chdir(VENDOR_PATH) do
   rescue Errno::ENOENT
     puts "Yarn executable was not detected in the system."
     puts "Download Yarn at https://yarnpkg.com/en/docs/install"
+    exit false
   end
```

Comment:
This change appears to cause a conflict with the yarn binstub that's created by `rails new`.  If the check in `install.rake` works out, then I think this change could easily be discarded.